scripts/build-all.ts

```diff
@@ -3,7 +3,8 @@
 import execa, { ExecaReturnValue } from 'execa';
 import { paths, getComponentPaths } from './paths';
 
-const build = (dir: string) => {
+const build = (dir?: string) => {
+	if (!dir) return;
 	console.log(`\nBuilding ${require(`${dir}/package.json`).name}`);
 
 	return execa('yarn', ['--cwd', dir, 'run', 'build'], {
```

Comment:
[`execa.sync`](https://github.com/sindresorhus/execa#execasyncfile-arguments-options) might help here?

then we should be able to do somtehing like `[...prioritisedPackages, ...packages].forEach(build)` i think?

Reply:
Ooh yeah, that'd be nicer. 

I added the `if (!dir) return;` because I was getting a type error about dir being `String | undefined` when I ran it but not in the editor and this fixed it. Not ideal but hopefully this script will be going soon anyway.

Reply:
👍